stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -80,4 +332,54 @@ private static String capitalize(String str) {
     buffer.setCharAt(0, Character.toTitleCase(buffer.charAt(0)));
     return buffer.toString();
   }
+
+  private class FieldBackedCSSProperty extends ViewCSSProperty {
+    private Field mField;
+
+    public FieldBackedCSSProperty(Field field, @Nullable ViewDebug.ExportedProperty annotation) {
+      super(field.getName(), annotation);
+      mField = field;
+      mField.setAccessible(true);
+    }
+
+    @Override
+    Object getValue(View view) throws InvocationTargetException, IllegalAccessException {
+      return mField.get(view);
+    }
+  }
+
+  private class MethodBackedCSSProperty extends ViewCSSProperty {
+    private Method mMethod;
+
+    public MethodBackedCSSProperty(Method method, @Nullable ViewDebug.ExportedProperty annotation) {
+      super(method.getName(), annotation);
+      mMethod = method;
+      mMethod.setAccessible(true);
+    }
+
+    @Override
+    Object getValue(View view) throws InvocationTargetException, IllegalAccessException {
+      return mMethod.invoke(view);
+    }
+  }
+
+  private abstract class ViewCSSProperty {
+    private String mName;
+    private ViewDebug.ExportedProperty mAnnotation;
+
+    public ViewCSSProperty(String name, @Nullable ViewDebug.ExportedProperty annotation) {
```

Comment:
Why do we allow the annotation to be null? I thought we only process things which have this annotation?